merchantapi/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline

```

Comment:
Why this is needed

Reply:
For mocking final methods/classes, not needed anymore -> removing